src/components/Bookmarks/Bookmarks.stories.js

```diff
@@ -0,0 +1,10 @@
+import Bookmarks from './Bookmarks'
+export default {
+  title: 'Components/Bookmarks',
+  component: Bookmarks,
+}
+
+const Template = args => <Bookmarks {...args} />
+
+export const Default = Template.bind({})
+Default.args = {}
```

Comment:
Since you've got two states of a bookmark you could show them here. If you are not using storybook you could just delete this and add it later if you want to